content/09.evolution.md

```diff
@@ -96,11 +96,45 @@ IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their
+A closely related virus was identified in Chinese horseshoe bats (_Rhinolophus sinicus_), but the sequence identity was only 88% with SARS-CoV-1 [@doi:10.1073/pnas.0506735102].
+Therefore, the species of origin for SARS-CoV-1 remains unresolved.
+
+In the case of SARS-CoV-2, early interest for the emergence of the pathogen turned to live-animal markets in Wuhan [@doi:10.1038/d41586-020-00548-w; @url:https://www.thestandard.com.hk/sections-news-print/215457/Ill-husband-and-wife--add-to-Wuhan-riddle].
+A large percentage of early patients had visited the Huanan seafood market in Wuhan, and next-generation sequencing of samples collected from nine patients, eight of whom had visited the market, revealed extremely high sequence identity (99.98%), indicative of rapid spread [@doi:10/ggjr43].
+The sequence of the viral pathogen collected from these patients was also compared to known zoonotic pathogens.
+In particular, genomic research quickly highlighted significant similarity (about 88% sequence identity) between SARS-CoV-2 and bat-derived SARS-like coronaviruses, namely bat-SL-CoVZC45 and bat-SL-CoVZXC21 [@doi:10/ggjr43].
+Other analyses have reported even greater similarity between SARS-CoV-2 and the bat coronavirus BatCoV-RaTG13, with shared sequence identity as high as 96.2% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036].
+Bats are well-established as a disease reservoir, including for RNA viruses [@doi:10.1016/j.tim.2014.12.004; @doi:10.1016/j.virusres.2015.05.006; @doi:10.1128/CMR.00017-06].
+This evidence therefore suggested that the virus may have emerged as a result of zoonotic transfer of a virus from bats to humans, with the wildlife trade considered a potential source of exposure.
+
+Nevertheless, some fragments of the genome differ between SARS-CoV-2 and RATG13 by up to 17%, suggesting a complex natural selection process.
+Additionally, SARS-CoV-2 is closely related (91.02%) to a novel coronaviruses identified in Malayan pangolins (_Manis javanica_) infected with a respiratory disease in October 2019 [@doi:10.1016/j.cub.2020.03.022].
+Although the genome-wide sequence identity was lower between SARS-CoV-2 and this pangolin virus than BatCoV-RaTG13, its particularly high similarity in the receptor binding domain (RBD) of the spike (_S_) gene with SARS-CoV-2 drew further attention [@doi:10.1038/s41586-020-2169-0; @doi:10.1016/j.cub.2020.03.022].
+The SARS-CoV-2 RBD differs from the pangolin coronavirus RBD by only one amino acid change [@doi:10.1016/j.cub.2020.03.022], and the sequence identity between the regions is 97.4% [@doi:10.1038/s41586-020-2169-0].
+Pangolins were therefore identified as a potential intermediate host of SARS-CoV-2 between bats and humans.
+
+However, data collected from May 2017 to November 2019 by a research team interested in tick-borne illnesses identified no bats or pangolins sold at these markets leading up to the emergence of COVID-19 [@doi:10.1038/s41598-021-91470-2].
```

Comment:
It is possible that a bats infect animals associated with these markets thereby increasing the likelihood of finding a bat-derived virus at one of these markers

Reply:
```suggestion
However, data collected from May 2017 to November 2019 by a research team interested in tick-borne illnesses identified no bats or pangolins sold at these markets leading up to the emergence of COVID-19 [@doi:10.1038/s41598-021-91470-2].
Additionally, endemic bat species are typically in hibernation at the time that SARS-CoV-2 emerged [@doi:10/ggjr43].
```

Reply:
Perfect